File: app/src/main/java/br/com/algorit/mangaalert/retrofit/MangaRepository.java
package br.com.algorit.mangaalert.retrofit;

import java.util.List;

import br.com.algorit.mangaalert.model.Manga;
import retrofit2.Call;
import retrofit2.http.GET;

public interface MangaRepository {

    @GET("manga/find-all")
    Call<List<Manga>> findAll();
}

File: app/src/main/java/br/com/algorit/mangaalert/retrofit/MangaRepository.kt
package br.com.algorit.mangaalert.retrofit

import br.com.algorit.mangaalert.roomdatabase.model.Manga
import br.com.algorit.mangaalert.roomdatabase.model.Novel
import retrofit2.Call
import retrofit2.http.GET

interface MangaRepository {
    @GET("manga/find-all")
    fun findAll(): Call<List<Manga>>

    @GET("novel/find-all")
    fun findAllNovel(): Call<List<Novel>>
}